composer.json

```diff
@@ -37,7 +37,8 @@
     },
     "config": {
         "allow-plugins": {
-            "infection/extension-installer": true
+            "infection/extension-installer": true,
+            "pestphp/pest-plugin": true
```

Comment:
This is the wrong place to do it as it changes the configuration for toolbox, not for its runtime.

For the correct way see examples for other tools, like phpcs:

https://github.com/jakzal/toolbox/blob/1ef9e2f185afb5a208f3af214a5b77459e68c2a6/resources/phpcs.json#L22C1-L24

Reply:
Thanks